helpers/util.js

```diff
@@ -0,0 +1,3 @@
+export function isCsvPath(path) {
+    return /\.csv$/.test(path);
```

Comment:
Het is correcter om hier de extensie van de file in de triplestore te checken (zie ook https://github.com/mu-semtech/file-service#properties)